### PR TITLE
Added text model metrics and minor improvements

### PR DESCRIPTION
**Pull request by ActiveChooN**
This PR contains:
- Model metrics screen for text-generation models
- Helpers for text-generation models parameters
- Open on HF button
- Very minor refactoring

| Text generation | Metrics |
|--------|--------|
| Screenshot 2024-12-05 at 14 55 29 | Screenshot 2024-12-05 at 15 00 09 |

### CODE REVIEW

File: lib/pages/text_generation/widgets/model_properties.dart [hunk 1]
```diff
@@ -40,12 +42,21 @@ class ModelProperties extends StatelessWidget {
...
                       value: nf.format(inference.temperature),
                     ),
                     ModelProperty(
                       title: "Top P",
+                      description: "Top P controls the diversity of the output by limiting the selection to a subset of the most probable tokens.",
                       value: nf.format(inference.topP),
                     ),
+                    if (inference.project!.isPublic) HyperlinkButton(
+                      child: const Row(children: [
+                        Text("View on Hugging Face"),
```
Comment (Radwan-Ibrahim) [inline lib/pages/text_generation/widgets/model_properties.dart:55]: Can we change this to match the UX?
image

File: lib/pages/text_generation/widgets/model_properties.dart [hunk 2]
```diff
@@ -40,12 +42,21 @@ class ModelProperties extends StatelessWidget {
                     ),
                     ModelProperty(
                       title: "Temperature",
+                      description: "Temperature controls the randomness of the output. Higher values mean more random outputs.",
```
Comment (Radwan-Ibrahim) [inline lib/pages/text_generation/widgets/model_properties.dart:45]: Can we add a tooltip next to slider as well?
image

**Comment by ArendJanKramer** [comment]
The class PerformanceTile doesn't seem to match UX either

**Comment by ArendJanKramer** [comment]
please rebase

**Pull request merged**